analyzer/tests/functional/ctu/test_ctu.py

```diff
@@ -211,6 +211,15 @@ def __check_ctu_analyze(self, output):
         self.assertIn("lib.c:3:", output)
         self.assertIn("[core.NullDereference]", output)
 
+        # We assume that only main.c has been analyzed with CTU and it involves
+        # lib.c during its analysis.
+        connections_dir = os.path.join(self.report_dir, 'ctu_connections')
+        connections_file = os.listdir(connections_dir)[0]
```

Comment:
Do we expect more files in this directory? Will the first file always be `main.c`? Should `lib.c` be checked here too?

Reply:
This folder contains files only for sources that involve other sources during CTU analysis. For analyzing `lib.c` no other sources are needed. Actually we can assert that the length of this is 1, I'll do that.